deploy/crds/hawtio_v1alpha1_hawtio_cr.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: hawt.io/v1alpha1
 kind: Hawtio
 metadata:
-  name: hawtio-online
+  name: fuse-console
```

Comment:
@redhatHameed: are we going to update the name for the upstream operator?

Reply:
@akoserwal we are not going to change the name of the upstream operator- actually I created the tls secret with name fuse-console-tls- etc already in a namespace so I changed here to create the deployment with the same name but will revet this back, thanks for highlighting this.